### DISCUSSION TITLE
Intiial nexrender test errors

### DISCUSSION THREAD

**Discussion by jprenc** [inlife/nexrender#984] [Q&A] [answered]
Hi there,

I've just started messing around with nexrender. Initially i'm getting this error. I can't find any documentation on ARG_UNKNOWN_OPTION
![image](https://github.com/inlife/nexrender/assets/167048144/bb663b4b-c791-4d14-9d65-1485eae3d532)

any suggestions or documentation would be amazing.

Cheers,
jack

**Comment by inlife** [comment, marked as answer]
Hey there,

Seems like you are missing `=` between the argument key "binary" and argument value "C:\Program Files\..."

Try making it like this: `--binary="C:\\Program Files\\...."`

**Reply by jprenc** [reply]
Thanks @inlife. that fixed it. I'm now getting errors based on the footage-swapping when I trigger the render 
![image](https://github.com/inlife/nexrender/assets/167048144/5c29cde7-f710-450a-bdbf-d40709607976)

**Reply by inlife** [reply]
Hey there,
Seems like you are missing one `/` for the asset path, it should be `file:///d/....`
